packages/twenty-server/src/engine/api/graphql/workspace-query-builder/factories/args-string.factory.ts

```diff
@@ -55,22 +60,39 @@ export class ArgsStringFactory {
     return argsString;
   }
 
-  private buildStringifiedObject(
-    key: string,
-    obj: Record<string, any>,
+  private buildStringifiedObjectFromArray(
+    keyValuePairArray: Array<Record<string, any>>,
   ): string {
     // PgGraphql is expecting the orderBy argument to be an array of objects
-    if (key === 'orderBy') {
-      const orderByString = Object.keys(obj)
-        .sort((_, b) => {
-          return b === 'position' ? -1 : 0;
-        })
-        .map((orderByKey) => `{${orderByKey}: ${obj[orderByKey]}}`)
-        .join(', ');
```

Comment:
Why not keeping the same logic? Even if you manage an array, this should still work no?
1. Sort array element: position at the end, others do not move
2. Stringify